fedimint-server/src/consensus/server.rs

```diff
@@ -77,6 +78,8 @@ pub struct ConsensusServer {
     self_id_str: String,
     /// Just a string version of peer ids for performance
     peer_id_str: Vec<String>,
+    connection_status_channels: Arc<RwLock<BTreeMap<PeerId, PeerConnectionStatus>>>,
```

Comment:
Oh, I see you did put it into self, OK. But I'd at least make a `type ConnectionStatuses = Arc<RwLock<BTreeMap<PeerId, PeerConnectionStatus>>>`.

Reply:
BTW. I guess there's no `_channels` anywhere here anymore.

Reply:
I don't like new types since its more context to keep in my head.